docs/whatsnew/6.0.rst

```diff
@@ -22,6 +22,29 @@ By the numbers:
 * X pull requests have been merged since v5.3
 * X distinct people have contributed code
 
+.. _whatsnew-6.0-geodetic-representation-geometry:
+
+Define Geodetic Representations via their geometric parameters
+==============================================================
+
+The `~astropy.coordinates.BaseGeodeticRepresentation` now exposes its geometric
```

Comment:
I think this first sentence is a bit of an "implementation detail" that users do not really have to care about. Maybe:
```
The user may now define custom spheroidal models for the Earth or other planetary
bodies by subclassing `~astropy.coordinates.BaseGeodeticRepresentation` and defining
``_equatorial_radius`` and ``_flattening`` attributes::